### PR TITLE
update: jest.config.js, remove packages deleted from last release

### PR DESCRIPTION
**Pull request by bobbyu99**
#### Description of changes
Remove packages deleted from last release in jest.config.js. Before this change, it was causing some issue on Jest, specifically running an individual test by clicking the green triangle next to a test case will cause Jest to error. 

I ran `yarn extract-dependency-licenses` for this PR as well, and it seems to move a license info up in dependency_licenses.txt. 

#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [ ] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/master/CONTRIBUTING.md#tests)
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] New AWS SDK calls or CloudFormation actions have been added to relevant test and service IAM policies
- [ ] Any CDK or CloudFormation parameter changes are called out explicitly

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: dependency_licenses.txt
Comment (palpatim) [inline dependency_licenses.txt:1]: Why is there a change to this file? Can you confirm that you rebased against `main` before submitting this PR?

Comment (bobbyu99) [reply dependency_licenses.txt:1]: i undo the commit, rebase again, and ran `yarn extract-dependency-licenses`, it's still updating that file. not sure whats causing it. 
Screenshot 2024-08-21 at 4 40 07 PM

Comment (bobbyu99) [reply dependency_licenses.txt:1]: the text is identical, seems like `yarn extract-dependency-licenses` is moving it around

Comment (palpatim) [reply dependency_licenses.txt:1]: Thanks for confirming.

**Review by palpatim — APPROVED** [review]

**Review by dpilch — APPROVED** [review]

**Pull request merged**